backends/tc/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+#******************************************************************************
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#*****************************************************************************/
+
+set(P4TC_BACKEND_SOURCES
+    backend.cpp
+    ebpfCodeGen.cpp
+    introspection.cpp
+    midend.cpp
+    pnaProgramStructure.cpp
+    tc.cpp
+    tcAnnotations.cpp
+    version.cpp
+    ../ebpf/ebpfBackend.cpp
```

Comment:
There are now 3 back ends that use this ebpf code. At some point it might make sense to consolidate it.